### DISCUSSION TITLE
Public api /status for checking webserver/database health

### DISCUSSION THREAD

**Discussion by lavaplanetmayonnaise** [umami-software/umami#1485] [Ideas]
Hi,

It would be nice if we can have a public api (/status, /health, ...), returning only a simple code 200.
The api could do a select count() on a umami table (or something less ressource consumming), so we can check the whole stack : webserver and database.

It's particulary usefull for the supervision, especially when using a reverse proxy and a clustered architecture.

Thank you.

**Comment by briancao** [comment]
In 1.38.0, we've added an api/heartbeat endpoint that returns a 200.  We'll consider adding a select count in the next release.
